beamer/tests/agent/test_challenge.py

```diff
@@ -250,16 +249,6 @@ def test_challenge_4(request_manager, fill_manager, token, config):
 def test_challenge_5(request_manager, fill_manager, token, config, honest_claim):
     requester, target = alloc_accounts(2)
     (charlie,) = alloc_whitelisted_accounts(1, {request_manager, fill_manager})
-    proxy_l2b = HTTPProxy(config.rpc_urls["l2b"])
-    proxy_l2b.delay_rpc({"eth_getLogs": 3})
-    proxy_l2b.start()
-
-    l2b_rpc_url = "http://%s:%s" % (
-        proxy_l2b.server_address[0],
-        proxy_l2b.server_address[1],
-    )
-
-    config.rpc_urls["l2b"] = l2b_rpc_url
```

Comment:
Why is this no longer necessary?

Reply:
> Why is this no longer necessary?

I see no reason why that particular test should depend on the fact that `eth_getLogs` calls are delayed.
The test checks whether the agent correctly waits until (constant) fill wait time has passed, which is something determined by the state machine and must not be affected by the RPC performance.